puppeteer/package/agama-integration-tests.spec

```diff
@@ -47,6 +47,12 @@ outside.
 rm -f package-lock.json
 local-npm-registry %{_sourcedir} install --omit=optional --with=dev --legacy-peer-deps || ( find ~/.npm/_logs -name '*-debug.log' -print0 | xargs -0 cat; false)
 
+# node_modules cleanup
+%{_builddir}/agama/node-prune.sh
```

Comment:
should we run it also for agama web directory?

Reply:
Agama-web uses webpack which already bundles only the needed files into the target JS file. It would not help there.

Reply:
And we do obfuscation and minification as well...